src/Administration/Resources/app/administration/src/module/sw-product/snippet/en-GB.json

```diff
@@ -43,13 +43,14 @@
       "labelPurchaseUnit": "Selling unit",
       "labelReferenceUnit": "Basic unit",
       "labelPackUnit": "Packaging unit",
+      "labelPackUnitPlural": "Packaging unit plural",
       "placeholderTaxRate": "Select tax rate...",
       "placeholderPurchasePriceGross": "Enter purchasing price...",
       "placeholderPriceGross": "Enter gross price...",
       "placeholderPriceNet": "Enter net price...",
       "placeholderPurchaseUnit": "Enter selling unit...",
       "placeholderReferenceUnit": "Enter basic unit...",
-      "placeholderPackUnit": "Enter packaging unit...",
```

Comment:
You removed the singular placeholder here. Please revert this deletion.

Reply:
I have already make the changes in a merge :)